src/Microsoft.AspNetCore.Http.Features/project.json

```diff
@@ -21,6 +21,7 @@
     "netstandard1.3": {
       "dependencies": {
         "System.Collections": "4.0.11-*",
+        "System.ComponentModel": "4.0.1-*",
```

Comment:
FYI I moved this from `src/Microsoft.AspNetCore.Http.Abstractions/project.json`, same package that `interface` came from to this one.